infra/gcp/clusters/kubernetes-public/prow-build-test/main.tf

```diff
@@ -83,15 +83,15 @@ resource "google_service_account_iam_policy" "boskos_janitor_sa_iam" {
 }
 
 module "prow_build_test_cluster" {
-  source = "./k8s-infra-gke-cluster"
+  source = "../../modules/k8s-infra-gke-cluster"
```

Comment:
I was trying to denote that it's k8s-infra specific rather than intended to be a generic gke cluster usable outside of this repo.  But I think you're right.

Reply:
done in b834ccd